lib/SILGen/InitializeIdentity.h

```diff
@@ -0,0 +1,40 @@
+//===--- InitializeIdentity.h - dist actor ID init for SILGen -------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "Cleanup.h"
+
+namespace swift {
+
+class AllocStackInst;
+
+namespace Lowering {
+
+/// A clean-up designed to emit an initialization of a distributed actor's
+/// identity upon successful initialization of the actor's system.
+struct InitializeIdentity : Cleanup {
```

Comment:
minor: want to name it with something Distributed? we're fairly consistently naming all things/files with Distributed... something, maybe `InitializeDistributedActorIdentity`?

Reply:
oh yea good point. will do